src/guide/essentials/watchers.md

```diff
@@ -331,13 +331,17 @@ Tanto `watch` como `watchEffect` nos permiten producir efectos secundarios de fo
 
 Cuando mutas el estado reactivo, puede desencadenar tanto las actualizaciones de los componentes de Vue como los callbacks de los watchers creados por ti.
 
-De forma predeterminada, los callbacks del watcher creados por el usuario son llamados **antes** de las actualizaciones del componente Vue. Esto significa que si intentas acceder al DOM dentro de un callback de vigilancia, el DOM estará en el estado antes de que Vue haya aplicado cualquier actualización.
+Similar a las actualizaciones de componentes, las devoluciones de llamada de los watchers creados por el usuario se agrupan para evitar invocaciones duplicadas. Por ejemplo, probablemente no queremos que un watcher se dispare mil veces si agregamos sincrónicamente mil elementos a un arreglo que está siendo observado
 
-Si quieres acceder al DOM en un callback de vigilancia **después** de que Vue lo haya actualizado, debes especificar la opción `flush: 'post'`:
+Por defecto, la devolución de llamada de un watcher se llama *después* de las actualizaciones del componente padre (si las hay) y *antes* de las actualizaciones del DOM del componente propietario. Esto significa que, si intentas acceder al propio DOM del componente propietario dentro de la devolución de llamada del watcher, el DOM estará en un estado previo a la actualización.
```

Comment:
Aquí igual con el tema del Callback.

¿Componente propietario? No lo siento natural esta traducción. ¿Cómo podríamos darle sentido a la frase "and **before** the owner component's DOM updates"?

@David-Pena @glopzel

Reply:
Que tal?
 
"y antes de que se actualice el DOM del componente que lo posee."

Reply:
Traducido, quedo excelente esa traducción